### DISCUSSION TITLE
HI, could i configure logical backup parameters?

### DISCUSSION THREAD

**Discussion by AndriyVys** [zalando/postgres-operator#1960] [Q&A]
Hi, could i configure logical backup parameters? like
 logical_backup_job_prefix: l
  logical_backup_provider: 
  logical_backup_s3_bucket
  logical_backup_s3_endpoint
  logical_backup_s3_region
  logical_backup_s3_sse
on Cluster Manifest level, not on ConfigMap

**Comment by ErikLundJensen** [comment]
That is currently only configurable in the operator manifest. There is an open issue for this if you do a search for it.
